src/features/pastelDB/__tests__/pastelDBLib.test.ts

```diff
@@ -1,4 +1,6 @@
 import initSqlJs, { Database, QueryExecResult } from 'sql.js'
+import fs from 'fs'
+import { electron, log } from '../../../../__mocks__/electron'
```

Comment:
You don't need to import this anymore even, jest will auto mock it for you. Just import it directly from electron. The import will be automatically modified from `__mocks__` directory Refer here: https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/features/deepLinking/__tests__/deepLinking.test.ts